Source/SwiftLintFramework/Rules/Lint/NSObjectPreferIsEqualRule.swift

```diff
@@ -0,0 +1,68 @@
+import SourceKittenFramework
+
+public struct NSObjectPreferIsEqualRule: Rule, OptInRule, ConfigurationProviderRule, AutomaticTestableRule {
```

Comment:
I think we could promote this to a default rule, given how easy it's to make a mistake (I've definitely made this mistake several times)